data/map planets.txt

```diff
@@ -2504,6 +2561,29 @@ planet Luna
 		threshold 7500
 		fleet "Large Republic" 10
 
+planet "Mordente-Bridi"
+	attributes ccor military station urban
+	landscape land/city3
+	music ambient/machinery
+	description `A testament to the ingenuity of the engineering corps of the now defunct Confederated Councils of the Outer Rim, the massive Mordente-Bridi Station is one of the largest in human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest alloy the CCOR could manufacture at scale. Bridi Cylinder is pressurized and contains crew quarters and logistical infrastructure amounting to that of a sizable city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
```

Comment:
```suggestion
	description `A testament to the ingenuity of the engineering corps of the now defunct Confederated Councils of the Outer Rim, Mordente-Bridi is an O'Neill Cylinder and the largest station in Human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest alloy the CCOR could manufacture at scale. Bridi Cylinder is pressurized and contains crew quarters and logistical infrastructure amounting to that of a sizable city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
```

Reply:
I think we should be keeping the "is one of" line. We don't have any strict "is the biggest station" in human space, and we should probably do our best to keep it abstract.